test/test_jit_cuda_fuser.py

```diff
@@ -690,6 +760,19 @@ def t_float(x: torch.Tensor, y: torch.Tensor):
             self.assertEqual(o, jit_o)
             self.assertGraphContains(t_jit.graph_for(x, y, z), FUSION_GUARD)
 
+        elif category2 != "scalar":  # only test the case where first is scalar
```

Comment:
If we are skipping on:
`category1 == "scalar" and category2 != "scalar"`, maybe we should return earlier at https://github.com/csarofeen/pytorch/pull/1415/files#diff-044090d79d37e42b5f19593ffc112d1b37e4f815f1d669a02d2b4a19fa7a64b3R685-R686

Reply:
Updated. Thanks!